www/source/partials/docs/_dev-pkgs-composite-pkg.html.md.erb

```diff
@@ -0,0 +1,121 @@
+
+~~~
+pkg_origin="yourorigin"
+pkg_name="composite-example"
+pkg_type="composite"
+pkg_version="0.1.0"
+
+pkg_services=(
+    origin/package1
+    origin/package2
+)
+
+pkg_bind_map=(
+    [origin/package2]="http:origin/package1"
+)
+~~~
+
+The `pkg_origin`, `pkg_name`, and `pkg_version` settings serve the same role they do in other Habitat packages, but `pkg_type`, `pkg_services`, and `pkg_bind_map` are only specific to composite packages. 
+
+Plans for composite packages must have the `pkg_type` setting set to the string `"composite"`. Without `pkg_type` defined, the package is assumed to be a "standalone" package type, which is the default type for non-composite packages. 
+
+The `pkg_services` array is where you enumerate all the services that are in your composite package. Services are defined as having a binary that gets executed via a `run` hook. By default, these should be either packages you have built and/or installed on your local workstation, or packages that are in the `stable` channel in Builder. If you wish to use packages that are in the `unstable` channel, specify `HAB_BLDR_CHANNEL=unstable` in your environment when you build your composite package.
+
+> **Note** You must specify at least two services in the `pkg_services` array. 
+
+The last composite-specific setting, `pkg_bind_map`, lets you declare all the binding relationships between all the services in your composite package. In the example above, `origin/package2` has a bind named `http` that is satisfied by the `origin/package1` service. Additional binds can be added by separating them with spaces, for example `"http:origin/package1 otherbind:origin/package3"`.
+
+## Using Composite Packages
+To use a composite package:
+
+* Build it in Builder or in a local Studio environment as you would any other package
+* [Upload and promote it](/docs/developing-packages/#sharing-pkgs) if needed
+* Run `hab svc load` 
+* Run `hab svc start`
+
+> **Note** As with standalone packages, you can also reference a local composite package file directly when running the `hab svc load` subcommand. 
+
+Loading a composite and running it as a service will download and install all referenced services, if needed. Specific versions of the services used during the composite package build will be used when the composite package is loaded, started, or installed.
+
+~~~
+$ hab svc load cm/composite-example --channel unstable
+» Installing cm/composite-example from channel 'unstable'
+→ Using cm/composite-example/0.1.0/20171005220452
+★ Install of cm/composite-example/0.1.0/20171005220452 complete with 0 new packages installed.
+hab-sup(MN): The cm/composite-example-api-proxy service was successfully loaded
+hab-sup(MN): The cm/sample-node-app service was successfully loaded
+hab-sup(MN): The cm/composite-example composite was successfully loaded
+~~~
+
+The update strategy, Builder URL, update channel, application environment, service group, and topology are all shared by members of the composite.
```

Comment:
Might say that they're _currently_ shared by members of the composite; it's possible we'll tweak that a bit before we're done. If nothing else, it can help reinforce the evolving nature of this preview feature.